data-prepper-logstash-configuration/src/test/java/org/opensearch/dataprepper/logstash/mapping/OpenSearchPluginAttributesMapperTest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.logstash.mapping;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasKey;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.opensearch.dataprepper.logstash.mapping.OpenSearchPluginAttributesMapper.LOGSTASH_OPENSEARCH_INDEX_ATTRIBUTE_NAME;
+
+import org.junit.jupiter.api.Test;
+import org.opensearch.dataprepper.logstash.model.LogstashAttribute;
+import org.opensearch.dataprepper.logstash.model.LogstashAttributeValue;
+import org.opensearch.dataprepper.logstash.model.LogstashValueType;
+
+import java.util.Collections;
+import java.util.Map;
+
+class OpenSearchPluginAttributesMapperTest {
+
+    private OpenSearchPluginAttributesMapper createObjectUnderTest() {
+        return new OpenSearchPluginAttributesMapper();
+    }
+
+    @Test
+    void convert_logstash_index_date_time_pattern() {
+        final String dataPrepperIndexAttribute = "index";
```

Comment:
You can make this a constant in the class and use it elsewhere. It will make the code more readable since it will be clear that it is a constant.